components/form/ApplicantForm.tsx

```diff
@@ -1,61 +1,155 @@
+                className="form-control"
+                aria-label="Last Name"
+                disabled={loading}
+                {...register('lastName', { required: true, setValueAs: trimInput, validate: validateName })}
+                required
+              />
+              {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+              <label htmlFor="lastName">Last Name</label>
+              {errors.lastName && (
+                <p role="alert" className="d-block invalid-feedback text-start mb-0">
+                  {errors.lastName.type === 'required' && 'Last name is required'}
+                  {errors.lastName.type === 'validate' && 'Last name cannot contain special characters such as ^!#$%*=<>;{}"'}
+                </p>
+              )}
+            </div>
           </div>
         </div>
-        <div className="col-md-6 pb-2">
+
+        <div className="pb-2">
           <div className="form-floating">
-            <input name="lastName" type="text" className="form-control" aria-label="Last Name" disabled={loading} required />
+            <input
+              type="email"
+              className="form-control"
+              aria-label="email"
+              disabled={loading}
+              {...register('email', {
+                required: true,
+                pattern: EMAIL_VALIDATION_REGEX,
+                setValueAs: trimInput,
+              })}
+              required
+            />
             {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-            <label htmlFor="lastName">Last Name</label>
+            <label htmlFor="email">Email Address</label>
+            {errors.email && (
+              <p role="alert" className="d-block invalid-feedback text-start mb-0">
+                A valid email address is required
+              </p>
+            )}
           </div>
         </div>
-      </div>
 
-      <div className="pb-2">
-        <div className="form-floating">
-          <input name="email" type="email" className="form-control" aria-label="email" disabled={loading} required />
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="email">Email Address</label>
+        <div className="pb-2">
+          <div className="form-floating">
+            <input
+              type="date"
+              className="form-control"
+              aria-label="Date of birth"
+              disabled={loading}
+              {...register('dob', {
+                required: true,
+                validate: validateMinAge,
+              })}
+              required
+            />
+            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+            <label htmlFor="dob">Date of Birth</label>
+            {errors.dob && (
+              <p role="alert" className="d-block invalid-feedback text-start mb-0">
+                Only users above 18 years old are allowed
```

Comment:
I think "at least" is easier to understand here.

So maybe change "Only users above 18 years old are allowed" to "Sorry, we can only approve people who are at least 18 years old."

Reply:
Thanks, fixed I replaced "approve" by "verify"